### PR TITLE
Documentation for covariance estimators

### PR DESCRIPTION
**Pull request by qbarthelemy**
Classes `Covariance`, `ERPCovariances`, `XdawnCovariances`, `HankelCovariances` have a parameter `estimator`, with the same description "For a complete list of estimator, see utils.covariance." But module `utils.covariance` does not provide a documentation about supported estimators.

This PR solves #97, adding the documentation of function `utils.covariance.covariances`, and making cross-references to it.

### CODE REVIEW

File: pyriemann/spatialfilters.py [hunk 1]
```diff
@@ -268,18 +270,20 @@ class CSP(BilinearFilter):
     References
     ----------
     [1] Zoltan J. Koles, Michael S. Lazar, Steven Z. Zhou. Spatial Patterns
-        Underlying Population Differences in the Background EEG. Brain
-        Topography 2(4), 275-284, 1990.
+    Underlying Population Differences in the Background EEG. Brain Topography
+    2(4), 275-284, 1990.
```
Comment (agramfort) [inline pyriemann/spatialfilters.py:274]: I would revert these changes and make it so that references can be cross-link within the docstrings. See an example in :

https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/svm/_classes.py#L635

you need the `..`  and then you can reference with `[1]_`

**Review by agramfort — COMMENTED** [review]
also I cannot check doc artifact as doc build failed.

**Review by agramfort — APPROVED** [review]

**Comment by qbarthelemy** [comment]
@agramfort , can you merge?

**Comment by agramfort** [comment]
done

I gave you write access to repo congrats :)

>
